lib/ui/hooks.dart

```diff
@@ -47,8 +47,10 @@ String _localeClosure() => window.locale.toString();
 @pragma('vm:entry-point')
 _LocaleClosure _getLocaleClosure() => _localeClosure;
 
+// BUG: https://github.com/flutter/flutter/issues/23246
+// Change back to List<String> when fixed
 @pragma('vm:entry-point')
-void _updateLocales(List<String> locales) {
+void _updateLocales(List<dynamic> locales) {
```

Comment:
This should no longer be necessary now that https://github.com/flutter/engine/commit/2707b7433874a8b03e4a1f7341d45b5cdcc8dfc6 has landed